### ISSUE TITLE
Bad, sad error for duplicate function qualifiers

### ISSUE THREAD

**Issue by MaryamZi** [ballerina-platform/ballerina-lang#25287]
**Description:**
$title.

**Steps to reproduce:**
```ballerina
transactional transactional function foo() {
    
}
```

ballerina-internal.log
```log
[2020-08-14 19:47:24,050] SEVERE {b7a.log.crash} - Can't find resource for bundle java.util.PropertyResourceBundle, key error.duplicate.object.method.qualifier 
java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key error.duplicate.object.method.qualifier
	at java.util.ResourceBundle.getObject(ResourceBundle.java:450)
	at java.util.ResourceBundle.getString(ResourceBundle.java:407)
	at io.ballerinalang.compiler.internal.diagnostics.DiagnosticMessageHelper.getDiagnosticMessage(DiagnosticMessageHelper.java:38)
	at io.ballerinalang.compiler.internal.diagnostics.SyntaxDiagnostic.message(SyntaxDiagnostic.java:68)
	at org.wso2.ballerinalang.compiler.parser.Parser.reportSyntaxDiagnostics(Parser.java:280)
	at org.wso2.ballerinalang.compiler.parser.Parser.generateCompilationUnitNew(Parser.java:131)
	at org.wso2.ballerinalang.compiler.parser.Parser.parseNew(Parser.java:117)
	at org.wso2.ballerinalang.compiler.PackageLoader.parse(PackageLoader.java:500)
	at org.wso2.ballerinalang.compiler.PackageLoader.loadEntryPackage(PackageLoader.java:380)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackages(Compiler.java:156)
	at org.wso2.ballerinalang.compiler.Compiler.compilePackage(Compiler.java:174)
	at org.wso2.ballerinalang.compiler.Compiler.compile(Compiler.java:91)
	at org.wso2.ballerinalang.compiler.Compiler.build(Compiler.java:99)
	at org.ballerinalang.packerina.task.CompileTask.execute(CompileTask.java:57)
	at org.ballerinalang.packerina.TaskExecutor.executeTasks(TaskExecutor.java:38)
	at org.ballerinalang.packerina.cmd.RunCommand.execute(RunCommand.java:299)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:57)
```

**Affected Versions:**
Current master

**Comment by MaryamZi** [comment]
Happens for the following too.

```ballerina
remote remote foo() {

}
```

Checked on slp3.

**Comment by SupunS** [comment]
Fixed with #25564
